File: app/src/main/java/com/soucriador/jhonattas/ui/activities/SplashScreenActivity.java
package com.soucriador.jhonattas.ui.activities;

import android.content.Intent;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.soucriador.jhonattas.R;

public class SplashScreenActivity extends AppCompatActivity implements Runnable {

    private final Long SPLASH_TIME_OUT = 3000L;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);

        new Handler().postDelayed(
                SplashScreenActivity.this,
                SPLASH_TIME_OUT
        );
    }


    @Override
    public void run() {
        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/soucriador/jhonattas/ui/activities/SplashScreenActivity.kt
package com.soucriador.jhonattas.ui.activities

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import com.soucriador.jhonattas.R

class SplashScreenActivity : AppCompatActivity(), Runnable {
    private val SPLASH_TIME_OUT = 1000L
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)
        Handler().postDelayed(
                this@SplashScreenActivity,
                SPLASH_TIME_OUT
        )
    }

    override fun run() {
        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}